ortools/dotnet/Google.sln

```diff
@@ -0,0 +1,76 @@
+
+Microsoft Visual Studio Solution File, Format Version 12.00
+# Visual Studio 15
+VisualStudioVersion = 15.0.26124.0
+MinimumVisualStudioVersion = 15.0.26124.0
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "OrTools", "OrTools\OrTools.csproj", "{AFA3F878-FB56-4314-B31E-3DA5583B548B}"
+EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "OrTools.Tests", "OrTools.Tests\OrTools.Tests.csproj", "{CB6482EF-BFD1-4512-A24C-3AF3C8C70D3D}"
+EndProject
+Project("{F2A71F9B-5D33-465A-A702-920D77279786}") = "OrTools.FSharp", "OrTools.FSharp\OrTools.FSharp.fsproj", "{113A3FA7-F58E-4392-B6E9-5750813BE5D3}"
+EndProject
+Project("{F2A71F9B-5D33-465A-A702-920D77279786}") = "OrTools.FSharp.Tests", "OrTools.FSharp.Tests\OrTools.FSharp.Tests.fsproj", "{06BDA79F-5C92-46A4-85DC-C0F5730CA495}"
+EndProject
+Global
+	GlobalSection(SolutionConfigurationPlatforms) = preSolution
+		Debug|Any CPU = Debug|Any CPU
+		Debug|x64 = Debug|x64
+		Debug|x86 = Debug|x86
+		Release|Any CPU = Release|Any CPU
+		Release|x64 = Release|x64
+		Release|x86 = Release|x86
+	EndGlobalSection
+	GlobalSection(SolutionProperties) = preSolution
+		HideSolutionNode = FALSE
+	EndGlobalSection
+	GlobalSection(ProjectConfigurationPlatforms) = postSolution
+		{AFA3F878-FB56-4314-B31E-3DA5583B548B}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
```

Comment:
Do we really need this magic "hash" ?

Is it univeral or dependent of your system (i.e. kind of watermarking)

Reply:
This hash is the project identifier. So long as projects are added using the toolchain, this shouldn't be a problem.

Reply:
ok found ! https://msdn.microsoft.com/en-us/library/bb165951.aspx
BTW Thanks for the explanation